resources/zib-HealthcareProvider-Location.xml

```diff
@@ -0,0 +1,120 @@
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-healthcareprovider-v3.4-2020EN" />
+    <uri value="https://zibs.nl/wiki/HealthcareProvider-v3.4(2020EN)" />
+    <name value="Zib HealthcareProvider-v3.4(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-contactinformation-v1.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/ContactInformation-v1.2(2020EN)" />
+    <name value="zib ContactInformation-v1.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Location" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Location" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Location">
+      <path value="Location" />
+      <short value="OrganizationLocation" />
+      <alias value="OrganisatieLocatie" />
+      <mapping>
+        <identity value="zib-healthcareprovider-v3.4-2020EN" />
+        <map value="NL-CM:17.2.9" />
+        <comment value="OrganizationLocation" />
+      </mapping>
+    </element>
+    <element id="Location.name">
+      <path value="Location.name" />
+      <short value="LocationName" />
+      <definition value="Name of the location." />
+      <alias value="LocatieNaam" />
+      <mapping>
+        <identity value="zib-healthcareprovider-v3.4-2020EN" />
+        <map value="NL-CM:17.2.8" />
+        <comment value="LocationName" />
+      </mapping>
+    </element>
+    <element id="Location.alias">
+      <path value="Location.alias" />
+      <short value="LocationNumber" />
+      <definition value="Number of the location Number, if a numerical location identification is used next to or instead of a location name." />
+      <comment value="There is a difference in datatype between the zib LocationNumber and this element, namely integer versus string. Also there is a cardinality mismatch of 0..1 versus 0..*. Receiving systems should be resilient enough to handle non-integer and repeating values." />
+      <alias value="LocatieNummer" />
+      <mapping>
+        <identity value="zib-healthcareprovider-v3.4-2020EN" />
+        <map value="NL-CM:17.2.10" />
+        <comment value="LocatieNummer" />
+      </mapping>
+    </element>
+    <element id="Location.telecom">
+      <path value="Location.telecom" />
+      <slicing>
+        <discriminator>
+          <type value="profile" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <short value="ContactInformation" />
+      <definition value="Root concept of the ContactInformation partial information model. This root concept contains all data elements of the ContactInformation partial information model." />

```

Comment:
Dit commentaar is overbodig

Reply:
Heel telecom en address weggehaald. Denk dat we dat in de nl-core-laag wel terug kunnen zetten.